addon/components/ag-grid.js

```diff
@@ -38,15 +39,21 @@ export default Component.extend({
 
   didInsertElement() {
     this._super(...arguments);
-    
+
     let defaults = {columnDefs: [], rowData: []};
+    const agGridOptions = typeof config != "undefined" ? config.agGrid : false;
     
     if (!this.get('gridOptions')) {
       this.set('gridOptions', defaults);
     }
 
     run.scheduleOnce('afterRender', () => {
+      if (agGridOptions && agGridOptions.useEnterprise) {
```

Comment:
You can also make sure the licenseKey has a value:

`if (agGridOptions && agGridOptions.useEnterprise && agGridOptions.licenseKey) {`